src/main/java/org/edumips64/utils/cli/ShowCommand.java

```diff
@@ -0,0 +1,46 @@
+package org.edumips64.utils.cli;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.Option;
+
+@Command(name = "show", resourceBundle = "CliMessages")
+public class ShowCommand implements Runnable{
```

Comment:
Would it be possible to print the help if the user invokes `show` with no arguments? Currently it gives no feedback, which could be confusing.

Reply:
Yup, changed it!